test/units/cli/test_galaxy.py

```diff
@@ -444,3 +450,123 @@ def test_template_ignore_similar_folder(self):
 
     def test_skeleton_option(self):
         self.assertEquals(self.role_skeleton_path, context.CLIARGS['role_skeleton'], msg='Skeleton path was not parsed properly from the command line')
+
+
+@pytest.fixture()
+def collection_skeleton(request):
+    name, skeleton_path = request.param
+
+    galaxy_args = ['ansible-galaxy', 'collection', 'init', '-c']
+
+    if skeleton_path is not None:
+        galaxy_args += ['--collection-skeleton', skeleton_path]
+
+    test_dir = tempfile.mkdtemp()
+    if not os.path.isdir(test_dir):
+        os.makedirs(test_dir)
+    galaxy_args += ['--init-path', test_dir, name]
+
+    gc = GalaxyCLI(args=galaxy_args)
+    gc.run()
+
+    namespace_name, collection_name = name.split('.', 1)
+    collection_dir = os.path.join(test_dir, namespace_name, collection_name)
+    yield collection_dir
```

Comment:
What you really need is this:
```suggestion
    meta_path = collection_dir / 'galaxy.yml'
    with open(str(meta_path), 'r') as galaxy_meta:
        yield yaml.safe_load(galaxy_meta)
```

Reply:
I saw that `collection_dir` itself is used in a few tests, so maybe it makes sense to have a separate fixture just for reading the meta.

Reply:
There are only 2 tests that actually use this, I prefer the simple 2 lines for each test instead of having another fixture. The fixtures are already complex enough for people who are unfamiliar with it that I prefer not complicate it even more.

Reply:
But it's not a part of tests, having pre-requisites there is even more confusing. I think that not using framework-recommended tools just because somebody else may be too lazy to read the docs is not a proper excuse.